frontend/src/error/index.js

```diff
@@ -33,4 +40,20 @@ function handleError(statusCode) {
   return standardizedError.description
 }
 
+export function handleSnackbarError(statusCode) {
+  const standardizedError = standardizedErrors.find(
+    error => error.status === parseInt(statusCode)
+  )
+  if (!standardizedError) {
+    return {
+      message: "Something went wrong",
+      severity: SNACKBAR_SEVERITY.ERROR,
+    }
+  }
+  return {
+    message: standardizedError.description,
+    severity: standardizedError.severity,
+  }
+}
+
 export default handleError
```

Comment:
oo I dont remember this sneaking in, can you just export the handle error fn, we dont need a default export.

Reply:
Done.